s3_management/manage.py

```diff
@@ -278,7 +278,7 @@ def nightly_packages_to_show(self: S3IndexType) -> List[S3Object]:
             package_name = full_package_name.split('-')[0]
             package_build_time = extract_package_build_time(full_package_name)
             # Hard pass on packages that are included in our allow list
-            if package_name not in PACKAGE_ALLOW_LIST:
+            if package_name.lower() not in PACKAGE_ALLOW_LIST:
```

Comment:
Why not do that and keep historical names in place?
```suggestion
            if package_name.lower() not in PACKAGE_ALLOW_LIST.lower():
```